my_vacations.html

```diff
@@ -1,12 +1,23 @@
 <!DOCTYPE html>
 <html lang="fr">
     <head>
+<link rel="stylesheet" href="my_style.css">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<a href="my_vacations.html" class="active">Mes vacances</a>
```

Comment:
I'm not sure why there's this link here in the head. It appears on the page, above the lab. I think this line of code should be removed.